sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -850,3 +851,29 @@ ALTER TABLE `form_vitals` ADD `inhaled_oxygen_concentration` float(4,1) DEFAULT
 #EndIf
 
 UPDATE `list_options` SET `notes` = 'LOINC:11502-2' WHERE `list_options`.`list_id` = 'Clinical_Note_Type' AND `list_options`.`option_id` = 'laboratory_report_narrative';
+
+#IfMissingColumn patient_data care_team_status
+ALTER TABLE patient_data ADD COLUMN care_team_status VARCHAR(100) NULL DEFAULT NULL;
```

Comment:
@adunsulag ,
missing in database.sql

Reply:
another thing will change is using TEXT for this since patient_data has broken things in past because of the size of the fields (this is why everything we add to it now is basically TEXT to prevent breaking size limit especially for legacy users that are using lots more non-TEXT fields).

Reply:
care_team_status is the option_id FROM list_options.option_id for the care team status.  I would think the column types should match there.  I'll add a comment onto the field to clarify what it points to.  Also will add it into database.sql.

Reply:
note there are a large amount of option_id's placed in patient_data that are text. The problem with that table is that it has too many columns (back in the day they were all varchar instead of text and it broke openemr since it was too large when users added new entries in layout editor) and legacy users still have this issue (even going to utf8mb4 may of broken their installs). So, decided to just use text (note this table may someday become vertical if @sunsetsystems has his wishes, which means these will all be placed in a text field anyways like the layout based forms are).